recipes/gnuplot/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+./configure \
+    --prefix=$PREFIX \
+    --without-x \
+    --without-lua
```

Comment:
We have our own `readline` build, so we may want to use `--with-readline=` and add `readline` as a dep.